### ISSUE TITLE
【Question】: I want to join JakartaEE Working Group, but do I have to join Eclipse Foundation Member first?

### ISSUE THREAD

**Issue by zhengshundong01** [jakartaee/jakarta.ee#915]
i have read https://jakarta.ee/documents/membership/jakarta-ee-membership-prospectus.pdf

this pdf say：
learn how to become a member on this page：https://www.eclipse.org/membership/become_a_member/

i have see this page is Eclipse Foundation member，so i need join this first ?

**Comment by TanjaObradovic** [comment]
Yes, the requirement is to become  Eclipse Foundation member first, and we find that most companies start out as a Solution Member. I would like to encourage to start the process and our team will help you out the best option for your company. This is all provided you are asking about the corporate membership.

**Comment by edbratt** [comment]
Individuals who wish to make contributions of code, or participate as individual committer members of the working group only need an Eclipse User account with a valid Developer Certificate of Origin, part of the Eclipse Contributor Agreement form. User members only need an Eclipse User account (so that you can join mailing lists, submit bugs, etc.).

**Comment by TanjaObradovic** [comment]
Thanks @edbratt ! 
If this is a question for individual contribution, for an Eclipse User account please go here https://accounts.eclipse.org/user/register and for  Eclipse Contributor Agreement please look here https://www.eclipse.org/legal/ECA.php
If this is a corporate inquiry your starting point should be here https://accounts.eclipse.org/contact/membership/jakarta-ee
